src/services/software-delivery/index.html

```diff
@@ -0,0 +1,22 @@
+---
+layout: service_line
+title: Software Delivery | Quickly bring innovative ideas to market
+title-es: Software Delivery | Aporta tus ideas innovadoras al mercado rápidamente
+description: We can help you define and build the features you need by providing cross-functional teams or the extra capacity required to deliver at scale.
+description-es: Te ayudamos a definir y desarrollar las características que necesitas proporcionando equipos multifuncionales o la capacidad adicional requerida para hacer entregas a escala.
+image:
+   src: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
+metarobots: noindex,nofollow
+sitemap: false
+
+intro:
+  background-image-small: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
+  background-image-large: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
+  foreground-image-large: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
+
+curated_publications:
+location: 'soft-mod'
```

Comment:
What do you think about changing this location to `software-delivery-page` or something similar?

We would then have to add that location to the posts that we want to show in this location but I think it keeps it more understandable if each page is it's own location.